integration-enabler-java/src/main/java/com/ca/mfaas/eurekaservice/client/impl/ApiMediationClientImpl.java

```diff
@@ -38,17 +37,22 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import static com.ca.mfaas.constants.EurekaMetadataDefinition.*;
+
 public class ApiMediationClientImpl implements ApiMediationClient {
     private static final Logger log = LoggerFactory.getLogger(ApiMediationClientImpl.class);
```

Comment:
This can be removed (log is not used), together with 
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

Reply:
That's right. It was used but later the exception was re-thrown and handle in calling location/s. 
Removed.

Reply:
Done, removed logger and corresponding imports.